src/components/AvailableMentors/AvailableMentors.js

```diff
@@ -0,0 +1,588 @@
+import React from "react";
+import withAuthorization from "../WithAuthorization";
+import classNames from "classnames";
+
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import SearchIcon from "@material-ui/icons/Search";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TablePagination from "@material-ui/core/TablePagination";
+import TableRow from "@material-ui/core/TableRow";
+import TableSortLabel from "@material-ui/core/TableSortLabel";
+
+import Paper from "@material-ui/core/Paper";
+import Tooltip from "@material-ui/core/Tooltip";
+import { lighten } from "@material-ui/core/styles/colorManipulator";
+import Button from "@material-ui/core/Button";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import Radio from "@material-ui/core/Radio";
+import { getAvailableMentors } from "../../firebase/operations";
+import Snackbar from "@material-ui/core/Snackbar";
+import SnackbarContentWrapper from "../SnackbarContentComponent/SnackbarContentComponent";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import TextField from "@material-ui/core/TextField";
+
```

Comment:
If we are not using className, Link and all Dialog imports in the component, we can delete them.